### PR TITLE
Smooth Scroll on Moderation

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
- Adds smooth Scrolling

## How do I test this PR?
- Go to moderation
- Press J or K and move
- It should be smooth

### CODE REVIEW

File: plugins/talk-plugin-permalink/client/components/CopyButton.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  render () {
+    return (
+      <Button {...this.props} >
+        {t('common.copy')}
+      </Button>
+    );
+  }
+}
+
+export default withCopyToClipboard(ButtonCopyToClipboard);
```
Comment (alinhle) [inline plugins/talk-plugin-permalink/client/components/CopyButton.js]: Where is this module and file used?

File: plugins/talk-plugin-permalink/client/components/Permalink.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+              {copySuccessful && 'Copied'}
+              {copyFailure && 'Not supported'}
+            </Button>
+            
+          </div>
+        </div>
+      </ClickOutside>
+    );
+  }
+}
```
Comment (alinhle) [inline plugins/talk-plugin-permalink/client/components/Permalink.js]: We already have `talk/plugins/talk-plugin-permalink/client/components/PermalinkButton.js` that looks just like this file. Somethings odd.

**Comment by kgardnr** [comment]
THIS IS SO GREAT!!! 🎉

**Review by kgardnr — APPROVED** [review]

**Comment by okbel** [comment]
@cvle ready!

**Review by alinhle — APPROVED** [review]

**Pull request merged**